src/unity/python/turicreate/toolkits/object_detector/_sframe_loader.py

```diff
@@ -166,6 +170,9 @@ def is_valid(ann):
                     return ok_optional
 
                 # Unchanged boxes, for evaluation
+                if not hasattr(label, '__iter__'):
```

Comment:
Probably want to remove this before you check it in.

Reply:
Done.